### PR TITLE
[WEB-8] add @mudita/sm-cli guide

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [WEB-8]

**Description**
Add @mudita/sm-cli guide

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

**Self check**

- [x] Self CR'd
- [ ] Tests included
- [ ] Description and screenshots attached

**PR Status**

- [ ] Code Review
- [ ] Quality Assurance

**Blockers**

**Deploy Notes**

### CODE REVIEW

File: src/content/libraries.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Libraries
+
+This guide explaining how to install, use and configure the libraries developed by [@mudita](https://github.com/orgs/mudita/teams/web) web team.
+
+## Secrets Manager
+
+The secret's manager [@mudita/sm-cli](https://www.npmjs.com/package/@mudita/sm-cli) used to storing and sharing the environment variable between the developers.
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:7]:
```suggestion
The secrets manager [@mudita/sm-cli](https://www.npmjs.com/package/@mudita/sm-cli) is used to store and share the environment variables between developers. 
```

File: src/content/libraries.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Libraries
+
+This guide explaining how to install, use and configure the libraries developed by [@mudita](https://github.com/orgs/mudita/teams/web) web team.
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:3]:
```suggestion
This guide explains how to install, use and configure the libraries developed by [@mudita](https://github.com/orgs/mudita/teams/web) web team.
```

File: src/content/libraries.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Secrets Manager
+
+The secret's manager [@mudita/sm-cli](https://www.npmjs.com/package/@mudita/sm-cli) used to storing and sharing the environment variable between the developers. 
+The library provides a friendly CLI interface which helps developers to send their variables to [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) and protected them from not authorized access from outside.
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:8]:
```suggestion
The library provides a friendly CLI interface which helps developers to send their variables to [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) and protect them from unauthorised access from outside.
```

File: src/content/libraries.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Installation
+For star using `sm-cli` you should install the module globally via:
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:11]:
```suggestion
To star using `sm-cli` you should install the module globally via:
```

File: src/content/libraries.mdx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+"secretsmanager:RestoreSecret",
+"secretsmanager:RotateSecret",
+"secretsmanager:UpdateSecretVersionStage",
+"secretsmanager:TagResource"
+"secretsmanager:ListSecrets"
+```
+
+### Simple usage
+
+1. On AWS Secrets Manager dashboard click on `Store new secrets` for creating new secrets
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:52]:
```suggestion
1. On AWS Secrets Manager dashboard click on `Store new secrets` to create new secrets.
```

File: src/content/libraries.mdx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+$ sm-cli init
+  Enter project name: your_project_name
+```
+it should create `.secretsrc` file in root directory
+
+5. Patse you variables to `.env` file and run
+```bash
+$ sm-cli sync
+```
+This command will synchronize your local env with secrets manager storage
```
Comment (mateuszkocz) [inline src/content/libraries.mdx:66]:
```suggestion
This command will synchronise your local env with secrets manager storage.
```

File: src/pages/guides/index.tsx [hunk 7]
```diff
@@ -42,6 +42,13 @@ const GuidePage: AppFunctionComponent = () => {
             }
             link={"/guides/okr"}
           />
+          <Box
+            title={"Libraries"}
+            description={
+              "Learn how we install ans configure external libraries to improving our jobs."
```
Comment (mateuszkocz) [inline src/pages/guides/index.tsx:48]:
```suggestion
              "Learn how we install and configure external libraries to improve our jobs."
```

**Review by mateuszkocz — COMMENTED** [review]
Some editorial stuff

**Comment by mateuszkocz** [comment]
@igorbezsmertnyi Please ensure the PR passes all required checks before requesting a review.

**Comment by mateuszkocz** [comment]
@ulaulala @mkurczewski Please review the guide so we can be sure it's understandable for everyone. 🙏

File: src/content/libraries.mdx [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+running command...
+$ sm-cli (-v|--version|version)
+@mudita/sm-cli/0.0.4 darwin-x64 node-v13.8.0
+$ sm-cli --help [COMMAND]
+USAGE
+  $ sm-cli COMMAND
+...
+```
+
+### Configura AWS CLI
```
Comment (ulaulala) [inline src/content/libraries.mdx:29]:
```suggestion
### Configure AWS CLI
```

File: src/content/libraries.mdx [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+$ sm-cli init
+  Enter project name: your_project_name
+```
+it should create `.secretsrc` file in root directory
+
+5. Patse you variables to `.env` file and run
+```bash
+$ sm-cli sync
+```
+This command will synchronise your local env with secrets manager storage.
```
Comment (ulaulala) [inline src/content/libraries.mdx:66]:
```suggestion
This command will synchronize your local env with secrets manager storage.
```

**Review by mkurczewski — DISMISSED** [review]

**Review by ulaulala — APPROVED** [review]

**Review by mkurczewski — APPROVED** [review]

**Pull request merged**